pkgs/desktops/gnome-3/core/gnome-software/default.nix

```diff
@@ -29,13 +35,16 @@ stdenv.mkDerivation rec {
     gtk3 glib packagekit appstream-glib libsoup
     gsettings-desktop-schemas gnome-desktop
     gspell json-glib libsecret ostree
-    polkit flatpak fwupd
-    libxmlb gnome-online-accounts
+    polkit flatpak libxmlb gnome-online-accounts
+  ] ++ stdenv.lib.optionals withFwupd [
+    fwupd
   ];
 
   mesonFlags = [
     "-Dubuntu_reviews=false"
     "-Dgudev=false"
+  ] ++ stdenv.lib.optionals (!withFwupd) [
+    "-Dfwupd=false"
```

Comment:
Why this? Should not fwupd now build on aarch64?

Reply:
I only pushed this so I could see if it would build on the aarch64 builder, or there wasn't other things broken. Currently building fwupd needs ghc which samueldr informed my was broken for him.

Reply:
#66277